docs/enterprise-guide/data-sandboxes.md

```diff
@@ -136,15 +136,15 @@ I.e., if a user belongs to two user groups, both of which have been given sandbo
 
 You can't currently create data sandboxes with non-SQL databases such as Google Analytics, Apache Druid, or MongoDB .
 
-#### Data sandboxes do not work on SQL/native queries
+#### Data sandboxes do not work if users can execute SQL/native queries
 
-This goes for both saved or unsaved SQL queries. The reason is that Metabase does not currently parse the contents of SQL queries, and therefore can't know conclusively which table(s) are referenced in a query. There are three specific situations to be aware of when it comes to SQL and sandboxes:
+If you're using data sandboxing, your user groups should _not_ have access to the native SQL editor or to saved SQL queries. The reason is that Metabase does not currently parse the contents of SQL queries, and therefore can't know conclusively which table(s) are referenced in a query. There are three specific situations to be aware of when it comes to SQL and sandboxes:
```

Comment:
```suggestion
If you're using data sandboxing, your groups should _not_ have access to:

1. The native SQL editor for that data.
2. Any collections with saved SQL queries that use that data.

The reason is that Metabase doesn't currently parse the contents of SQL queries, and therefore can't know conclusively which table(s) are referenced in a query. 

There are three specific situations to be aware of when it comes to SQL and sandboxes:
```

Just trying to emphasize that we give groups access to collections, not questions.